plugins/modules/zos_blockinfile.py

```diff
@@ -96,13 +96,15 @@
   marker_begin:
     description:
     - This will be inserted at C({mark}) in the opening ansible block marker.
+    - Required unique value different from marker_end.
```

Comment:
Thinking about this again, I don't think we should use word "required" because it can mislead the user to think this option is required when in fact is not. 

How about changing to ? 


```suggestion
    - Value needs to be different from marker_end.
```